modules/optflow/src/rlof/berlof_invoker.hpp

```diff
@@ -1620,7 +1620,6 @@ class TrackerInvoker  : public cv::ParallelLoopBody
             nextPt += halfWin;
             Point2f prevDelta(0,0);
 
-            float c[8];
```

Comment:
Main problem is here.
Scope of this variable is unclear. See code near line 1800:

```
                if(hasSolved == false)
                {
                    delta = Point2f( c[0] * ab + c[1] * a + c[2] * b + c[3],
                                     c[4] * ab + c[5] * a + c[6] * b + c[7]);
```

/cc @tsenst